src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/SimpleTypeModelBinder.cs

```diff
@@ -47,23 +47,30 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             {
                 var value = valueProviderResult.FirstValue;
 
-                object model = null;
-                if (!string.IsNullOrWhiteSpace(value))
-                {
-                    model = _typeConverter.ConvertFrom(
-                        context: null,
-                        culture: valueProviderResult.Culture,
-                        value: value);
-                }
-
+                object model;
                 if (bindingContext.ModelType == typeof(string))
                 {
-                    var modelAsString = model as string;
-                    if (bindingContext.ModelMetadata.ConvertEmptyStringToNull &&
-                        string.IsNullOrEmpty(modelAsString))
+                    // Already have a string. No further conversion required but handle ConvertEmptyStringToNull.
+                    if (bindingContext.ModelMetadata.ConvertEmptyStringToNull && string.IsNullOrWhiteSpace(value))
```

Comment:
No, the `StringConverter` just returns a `string` source unchanged. It's about the only `TypeConverter` that does _not_ `Trim()`.

Note I did not have to change any existing tests except to remove a duplicate.